test/e2e-common.sh

```diff
@@ -1,4 +1,6 @@
-# Copyright 2022 The Knative Authors
+#!/usr/bin/env bash
+
+# Copyright 2018 The Knative Authors
 #
```

Comment:
copy from serving repo

Reply:
Let's keep the dates in both scripts as 2022

Reply:
fixed